src/components/molecules/plan/plan.html

```diff
@@ -0,0 +1,31 @@
+<div class="plan">
+  <div class="text">プランテキスト</div>
+  <div class="body">
+    <div class="price">
+      <span class="yen">￥</span>5,000
+      <p class="memo">月額</p>
+    </div>
+  </div>
+  <div class="list">
+    <p class="start">入会金0円</p>
+    <p class="start">平日:9時～17時</p>
+    <p class="start">土日祝:利用ナシ</p>
+    <p class="start">コアタイムプランでの利用に最適なプラン</p>
+  </div>
```

Comment:
並列ということを意識した場合
```suggestion
  <ul class="list">
    <li class="start">入会金0円</li>
    <li class="start">平日:9時～17時</li>
    <li class="start">土日祝:利用ナシ</li>
    <li class="start">コアタイムプランでの利用に最適なプラン</li>
  </ul>
```
が最適です...！

Reply:
同感です、さらにいうとクラス名も手を入れてもいいかもですね
`item` という単語とかは、今回のような、同質のものが複数ある場合につけられがちな気がします。